resources/lang/ru/time.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Russian translation of uptime and followage
+ * created by rhizoids of kriper1111
+ * can't figure out how to specify modulus (if even possible) (for example "1", "21", "31", "61", etc. should have the same translation)
```

Comment:
As commented in the pull request thread, this should work fine as it currently is, since Laravel should deal with it automatically.

(Can probably remove this comment)